src/test/java/com/puppycrawl/tools/checkstyle/utils/CheckUtilTest.java

```diff
@@ -443,6 +444,24 @@ public void testParseClassNames() {
             .isEqualTo(expected);
     }
 
+    @Test
+    public void testIsPackageInfo() throws IOException {
+        final String path = getPath("package-info.java");
+
+        assertWithMessage("Should return true when is package info")
+                .that(CheckUtil.isPackageInfo(path))
+                .isTrue();
+    }
+
+    @Test
+    public void testIsNotPackageInfo() throws IOException {
+        final String path = getPath("InputCheckUtil-package-info.java");
```

Comment:
Please remove such tests.
Only test we allow are by Inputs and execution of whole checkstyle

Reply:
Done